src/components/postDetail/reservationPanel/Calendar/index.tsx

```diff
@@ -0,0 +1,152 @@
+  const today = dayjs();
+
+  const [selectedYear, setSelectedYear] = useState(today.year());
+  const [selectedMonth, setSelectedMonth] = useState(today.month() + 1);
+  const [selectedDate, setSelectedDate] = useState(availableSchedule[0]?.date || '');
+
+  const { isVisible: arrowLeftHover, handleToggleClick: arrowLeftClick } = useToggleButton();
+  const { isVisible: arrowRightHover, handleToggleClick: arrowRightClick } = useToggleButton();
+
+  const { url: arrowLeftUrl, alt: arrowLeftAlt } = arrowLeftHover ? left.active : left.default;
+  const { url: arrowRightUrl, alt: arrowRightAlt } = arrowRightHover ? right.active : right.default;
+  const isAllSchedulesReserved = availableSchedule.length === 0;
+  const monthName = dayjs()
+    .month(selectedMonth - 1)
+    .format('MMMM');
+  const maxDate = today.add(30, 'day');
+
+  const updateAvailableTimes = (value: string) => {
+    setSelectedDate(value);
+    const selectedDateSchedule = availableSchedule.find((a) => a.date === selectedDate);
+    let updatedTimes: AvailableTimesOptions[] = [];
+
+    if (selectedDateSchedule) {
+      updatedTimes = selectedDateSchedule.times.map((item) => ({
+        value: item.id,
+        title: `${item.startTime}-${item.endTime}`,
+      }));
+    } else {
+      updatedTimes = [
+        {
+          title: '예약 가능한 시간대가 없습니다.',
+          value: 0,
+        },
+      ];
+    }
+
+    setAvailableTimes(updatedTimes);
+  };
+
+  const handlePrevMonthClick = () => {
+    if (selectedMonth === today.month() + 1 && selectedYear === today.year()) return;
+
+    setSelectedMonth((prevMonth) => {
+      const newMonth = prevMonth === 1 ? 12 : prevMonth - 1;
+      const newYear = prevMonth === 1 ? selectedYear - 1 : selectedYear;
+      setSelectedYear(newYear);
+
+      return newMonth;
+    });
+  };
+
+  const handleNextMonthClick = () => {
+    if (selectedMonth === maxDate.month() + 1 && selectedYear === maxDate.year()) return;
+    setSelectedMonth((prevMonth) => {
+      const newMonth = prevMonth === 12 ? 1 : prevMonth + 1;
+      const newYear = prevMonth === 12 ? selectedYear + 1 : selectedYear;
+      setSelectedYear(newYear);
+
+      return newMonth;
+    });
+  };
+
+  useEffect(() => {
+    updateAvailableTimes(selectedDate);
+    setIsNoSchedule(isAllSchedulesReserved);
+  }, [selectedDate, isAllSchedulesReserved]);
+
+  console.log(activityId);
+
+  return (
+    <article className={cx('calender')}>
+      <header className={cx('calender-header')}>
+        <div className={cx('calender-header-date')}>
+          {monthName} {selectedYear}
+        </div>
+        <div className={cx('calender-header-control')}>
+          <button
+            className={cx('calender-header-control-prev')}
+            onMouseEnter={arrowLeftClick}
+            onMouseLeave={arrowLeftClick}
+            onClick={handlePrevMonthClick}
+          >
+            <Image src={arrowLeftUrl} alt={arrowLeftAlt} width={16} height={16}></Image>
```

Comment:
```suggestion
            <Image src={arrowLeftUrl} alt={arrowLeftAlt} width={16} height={16} />
```
이렇게 사용할 수 있을 것 같습니다~

Reply:
어머!!!